recipe/meta.yaml

```diff
@@ -28,8 +29,10 @@ test:
 about:
   home: https://www.gnu.org/software/make/
   license: GPLv3
```

Comment:
The license should be `GPL-3.0-or-later`, see https://www.gnu.org/licenses/gpl-3.0.html
And add please `license_file: COPYING`, http://git.savannah.gnu.org/cgit/make.git/tree/COPYING